src/config/scrivitoContentBrowser.js

```diff
@@ -1,107 +1,129 @@
 import * as Scrivito from "scrivito";
 
 Scrivito.configureContentBrowser({
-  filters: {
-    _objClass: {
-      options: {
-        All: {
-          title: "All",
-          icon: "folder",
-          query: Scrivito.Obj.all(),
-          selected: true,
-        },
-        Images: {
-          title: "Images",
-          icon: "image",
-          field: "_objClass",
-          value: "Image",
-        },
-        Pages: {
-          title: "Pages",
-          icon: "sheet",
-          field: "_objClass",
-          value: [
-            "Author",
-            "Blog",
-            "BlogPost",
-            "Event",
-            "Homepage",
-            "Job",
-            "LandingPage",
-            "Page",
-            "Redirect",
-            "SearchResults",
-          ],
-          options: {
-            Page: {
-              title: "Standard pages",
-              icon: "sheet",
-            },
-            LandingPage: {
-              title: "Landing pages",
-              icon: "inbox",
-            },
-            BlogPost: {
-              title: "Blog posts",
-              icon: "pen",
-            },
-            Author: {
-              title: "Authors",
-              icon: "user",
-            },
-            Event: {
-              title: "Events",
-              icon: "cal",
-            },
-            Job: {
-              title: "Jobs",
-              icon: "suitcase",
-            },
-            Redirect: {
-              title: "Redirects",
-              icon: "link",
-            },
-            Homepage: {
-              title: "Homepage",
-              icon: "inbox",
-            },
-            Blog: {
-              title: "Blog",
-              icon: "pen",
+  filters: filterContext => {
+    if (filterContext._validObjClasses && filterContext._validObjClasses[0]) {
+      const validObjClasses = filterContext._validObjClasses;
+      const [head, ...tail] = validObjClasses;
+
+      if (tail.length) {
+        return {
+          _objClass: {
+            options: {
+              All: {
+                title: "All",
+                icon: "folder",
+                field: "_objClass",
+                value: validObjClasses,
+              },
+              [head]: {
+                ...descriptionForObjClass(head),
+                selected: true,
```

Comment:
I proposed to put this to the `All` section. There are pro and cons.

Pro:
- Your initially unfolded filter will not hide Objs you wanted to select.
- You do not depend on a "developer's choice".
- Would simplify code ... its only `reduce` then.

Con:
- More Objs displayed immediately.
- Your developer is not able to present you your favoured ObjClass first with a simple reorder of the restriction definition.
- It is easier to simplify the current code than to convert the simple `reduce` into the one with the specific `head`.